docs/website/docs/user-guides/quickstart/docs-mdx/odo_init_description.mdx

```diff
@@ -6,6 +6,10 @@ Now we'll initialize your application by creating a `devfile.yaml` to be deploye
 
 <p>Let's run <code>odo init</code> and select <span>{props.framework}</span>:</p>
 
+:::note
+When you first run `odo init`, it will detect the required devfile to be 'dotnet50', if this happens to you, please select <b>No</b> when asked <b>Is this correct?</b> and then select <b>.NET 6.0</b> when asked for <b>Select project type:</b>. Take a look at the sample output below for a reference.
+:::
```

Comment:
Adding the note here will affect all the other Quickstart Guides, not only the one for . NET.

Reply:
:facepalm: You're right.